src/Maker/MakeEntity.php

```diff
@@ -848,6 +860,26 @@ private function doesEntityUseAnnotationMapping(string $className): bool
         return $this->doctrineHelper->isClassAnnotated($className);
     }
 
+    private function doesEntityUseAttributeMapping(string $className): bool
```

Comment:
Instead of a new method, let's rename `doesEntityUseAnnotationMapping()` to `doesEntityUseAnnotationOrAttributeMapping()` and update it to check for both.

Reply:
Whats about `doesEntityUseSupportedMapping` method instead of using the Driver names in the method?

Reply:
That's a nice idea :)